src/components/Card/Card.css

```diff
@@ -22,4 +22,17 @@
 .results--artist {
   margin-top: 7px;
   margin-block-end: 0;
+}
+
+.delete-button {
+  color: #5518AB;
+  background-color: transparent;
```

Comment:
Colour and background should match the other buttons. This one is a little hard to read

Reply:
removed the buttons from card and now use the button in album details which changes on conditional if album is in or not in the Colletions.